octoprint_discordremote/command.py

```diff
@@ -25,6 +28,11 @@ def __init__(self, plugin):
         self.command_dict['/help'] = {'cmd': self.help, 'description': "Print this help."}
         self.command_dict['/pause'] = {'cmd': self.pause, 'description': "Pause current print."}
         self.command_dict['/resume'] = {'cmd': self.resume, 'description': "Resume current print."}
+        self.command_dict['/timelapse'] = {'cmd': self.timelapse, 'description': "List all timelapses."}
+        self.command_dict['/gettimelapse'] = {'cmd': self.get_timelapse, 'params': "[filename]",
+                                              'description': "Gets the download link for the specified timelapse."}
+        self.command_dict['/getfile'] = {'cmd': self.get_file, 'params': "[location] [filename]",
```

Comment:
@OctopusProteins Manditory params should be written like: {location} {filename}. Square brackets are for optional params.

Reply:
Fixed by 3991180
Also added error case if missing parameters